### ISSUE TITLE
Products Module - Can't Fill in Quantity in-stock

### ISSUE THREAD

**Issue by visitekasia** [YetiForceCompany/YetiForceCRM#4463]
<strong>Dear Friends,</strong><br />Why am I not able to fill in the quantity in-stock on "Products" module?<br />Is there something need to be done before filling in the quantity in-stock?<br /><img alt="" src="http://i.imgur.com/CeOgxDR.jpg" /><br />There is also another issue regarding to my PHP.ini.<br />I can't edit these 2 lines on my htaccess:<br />expose_php: On<br />innodb_lock_wait_timeout: 50<br /><br />I tried to insert these 2 lines on htaccess but it doesn't works:<br />php_flag expose_php off<br />php_value innodb_lock_wait_timeout 600<br /><br /><strong>Regards,<br />Nando</strong><br /><hr />Generated by YetiforceCRM ver. 4.0.0<br />PHP: 5.6.30<br /><br /><strong>Server configuration errors:</strong><br />expose_php: On<br />innodb_lock_wait_timeout: 50

**Comment by visitekasia** [comment]
**Dear Friends,**
I have found out how to add product quantity, it should be done from "Logistics > Goods Received Note".
Is there anyway for me to turn off the whole logistics module and make the quantity in-stock editable from products module?
And I want to make an order module that allow users to make orders for the products and the order quantity of the products automatically add the value of quantity in-demand on the products module.

**Regards,
Nando**
